client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -54,6 +69,41 @@ export default connectOptions(
 					<ThanksModal
 						site={ site }
 						source={ 'list' } />
+					{ config.isEnabled( 'manage/themes/upload' ) &&
```

Comment:
Let's not forget to also add a check for the ~~`eeMacarena`~~ [hasJetpackSiteJetpackThemesExtendedFeatures](https://github.com/Automattic/wp-calypso/blob/c770439cd70fe4183625a19bda50dc863f4dcae8/client/state/sites/selectors.js#L751) selector here in a follow-up PR. (Same for the other upload button on this page.)